website/source/guides/securing-nomad.html.md

```diff
@@ -469,16 +469,31 @@ tls {
 ```
 ## Migrating a cluster to TLS
 
-Nomad supports dynamically reloading it's TLS configuration. To reload Nomad's
-configuration, first update the configuration file and then send the Nomad
-agent a SIGHUP signal. Note that this will only reload a subset of the
-configuration file, including the TLS configuration.
+### Reloading TLS configuration via SIGHUP
+
+Nomad supports dynamically reloading both client and server TLS configuration.
+To reload an agent's TLS  configuration, first update the TLS block in the
+agent's configuration file and then send the Nomad agent a SIGHUP signal.
+Note that this will only reload a subset of the configuration file,
+including the TLS configuration.
 
 When reloading the configuration, if there is a change to the TLS
 configuration, the agent will reload all network connections and when
-establishing new connections, will use the new configuration. This process
-works for both upgrading and downgrading TLS (but we recommend upgrading).
-
+establishing new connections, will use the new configuration. The agent will
+also close any outstanding old connections. This process works for both
+upgrading and downgrading TLS (but we recommend upgrading).
```

Comment:
`upgrading and downgrading TLS (but we recommend upgrading), as well as rolling certificates.`